### DISCUSSION TITLE
Multiple Language Support for single recipe

### DISCUSSION THREAD

**Discussion by TJohansen** [mealie-recipes/mealie#1684] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [X] Reviewed the documentation and verified that my feature request isn't already a feature.
- [X] Reviewed the existing feature requests and my feature has not been requested.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
I save all my recipes in Norwegian for them to be shared with my Norwegian family and friends, but I also have a lot of non-Norwegian speaking friends so sometimes I go through the effort of translating a recipe for them before sharing it with them.

**(Optional): Suggest A Solution**
After going through the effort of translating a recipe I'd like to be able to save this as an English version of a recipe that can be swapped to without creating duplicate recipes.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by mjuddi** [comment]
Glad I found this, I have exactly the same idea.

I'm Icelandic but I have a German girlfriend and my brother has a Spanish girlfriend. I would love to be able to share recipes with them and their families and vice versa to get the best from Spanish / German / Icelandic cuisine.

It would be cool if there would be a language tab within the recipes, where you could flip between languages (not saying auto-translate or anything like that, just the option to be able to set more than one language per recipe without duplicating everything)

Other than that, I love this project, kudos hay-kot!

**Comment by alepouna** [comment]
Bumping this idea, if doesn't already exist in Mealie.

**Comment by Professor-64** [comment]
I need this idea so hard for my foreign friends. I like the suggestion to work with versions. An automatic tool would also be a great idea.
